### ISSUE TITLE
Why sync seed?

### ISSUE THREAD

**Issue by martinrebane** [mit-han-lab/spvnas#82]
Hi!
I guess `seed` is fixed in `train.py` to be able to repeat the results. But just to be sure - the `seed` is sync'ed for the same reason, syncing is not required to train a new model in SPVNAS or torchsparse? 

```
seed = configs.train.seed + dist.rank(
    ) * configs.workers_per_gpu * configs.num_epochs
    random.seed(seed)
    np.random.seed(seed)
    torch.manual_seed(seed)
    torch.cuda.manual_seed(seed)
```
If I remove fixed seed from config, I can also remove syncing (unless I need to reproduce the results later)?
Thanks!

**Comment by CCInc** [comment]
You can set any seed you like, it will only vary the results. You're right
that it's just fixed for reproducibility purposes.

**Comment by martinrebane** [comment]
Thanks for confirming!
